src/test/storage/encoded_column_test.cpp

```diff
@@ -109,6 +109,8 @@ INSTANTIATE_TEST_CASE_P(
     ColumnEncodingSpecs, EncodedColumnTest,
     ::testing::Values(ColumnEncodingSpec{EncodingType::Dictionary, VectorCompressionType::SimdBp128},
                       ColumnEncodingSpec{EncodingType::Dictionary, VectorCompressionType::FixedSizeByteAligned},
+                      ColumnEncodingSpec{EncodingType::FrameOfReference, VectorCompressionType::SimdBp128},
+                      ColumnEncodingSpec{EncodingType::FrameOfReference, VectorCompressionType::FixedSizeByteAligned},
```

Comment:
I would an an assertion somewhere that makes sure that the test size is big enough to fill at least three blocks of the default size.

Reply:
Okay

Reply:
But how?